test/hotspot/jtreg/runtime/cds/appcds/test-classes/pkg/ClassInPackage.java

```diff
@@ -0,0 +1,6 @@
+package pkg;
```

Comment:
Does this simple test class require a copyright header?

Reply:
Fixed.